### DISCUSSION TITLE
Different currency for the product

### DISCUSSION THREAD

**Discussion by gs2k84** [PrestaShop/PrestaShop#40149] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Hello,

I believe that a very important option would be the different currency option for the product.

Explanation.

When I buy goods in different countries, I pay in different currencies.

For example, when buying goods in France, I pay in EUR
I pay in GBP to buy goods in Great Britain
When buying goods in Poland, I pay in PLN.

### Describe the solution you'd like

If it was possible to enter the price of the goods in EUR when adding to the warehouse for products bought in France, for a product bought in Great Britain in GBP and for a product bought in Poland, enter in PLN, the problem of changing the exchange rate would be solved.

At this point, when converting the purchase from today into the default currency of the store, the price will be different in a month.

When I put the price in the purchase currency for the product today, for a month I will still get the same EUR, GBP and PLN as when I bought the product.

The price on Frontpage, on the other hand, will change according to the current currency exchange rate, however, it will be the same amount of EUR, GBP and PLN.

**Comment by hibatallahAouadni** [comment]
Hello @gs2k84 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you
